keyboards/pteron56/readme.md

```diff
@@ -0,0 +1,31 @@
+# Pteron56
+
+![%Pteron56%](https://user-images.githubusercontent.com/67011325/215351130-55bd1d58-3774-4123-8ee2-7352dfbc8ddd.jpg)
```

Comment:
please re-host this image on imgur, including the `h` before the .imageFormat
e.g. `https://i.imgur.com/zcauNwi.jpeg` -> `https://i.imgur.com/zcauNwih.jpeg`